app/models/nfl_sentiment_config.py

```
@@ -0,0 +1,411 @@
+        "comeback",
+        "amazing",
+        "incredible",
+        "outstanding",
+        "phenomenal",
+        "beast mode",
+        "money",
+        "perfect",
+        "flawless",
+        "unstoppable",
+        "legendary",
+    }
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)